lib/utils/ui/centauri_snack_bar.dart

```diff
@@ -0,0 +1,23 @@
+import "package:flutter/material.dart";
+
+const centauriPointsSnackBarDuration = Duration(seconds: 4); // default value
+
+/// the state ScaffoldMessenger.of(context)
+/// Show a snackbar telling the user they won [value] Centauri points.
+/// [scaffoldMessengerState] is the current state of the scaffolds,
+/// on which to display the snackbar. It can typially be found
+/// using [ScaffoldMessenger.of(context)].
+void showCentauriPointsSnackBar(
+  int value,
+  ScaffoldMessengerState scaffoldMessengerState,
+) {
+  scaffoldMessengerState.showSnackBar(
+    SnackBar(
+      behavior: SnackBarBehavior.floating,
+      duration: centauriPointsSnackBarDuration,
+      content: Text(
+        "You won $value Centauri! 🎉",
+      ),
```

Comment:
```suggestion
      ),
      showCloseIcon: true,
```

I suggest we add a close icon to this snack bar, it will make the message easier to dismiss once it is read.

Reply:
I'm not sure I agree, a close icon is not very pretty in my opinion. I'm not sure it is a big issue if the user cannot close it immediately, since this is a rare pop-up.

I can decrease the time this snackbar appears however, if you wish. Currently, it stays for 4 seconds, which is the default value.

Reply:
I understand, I didn't consider that this would be a rare popup since I was testing it over and over again. Let's leave it like this.